ios_aaa_auth_null_user

```diff
@@ -0,0 +1,8 @@
+ios_aaa_authentication { 'suppress null-username':
```

Comment:
Guess this was your test file, probably could be removed or put in with the other `examples/`